common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivity.java

```diff
@@ -178,13 +227,31 @@ protected void onStop() {
     protected void onSaveInstanceState(final Bundle outState) {
         super.onSaveInstanceState(outState);
         outState.putParcelable(KEY_AUTH_INTENT, mAuthIntent);
+        outState.putParcelable(KEY_RESULT_INTENT, mResultIntent);
         outState.putBoolean(KEY_AUTHORIZATION_STARTED, mAuthorizationStarted);
         outState.putBoolean(KEY_PKEYAUTH_STATUS, mPkeyAuthStatus);
         outState.putSerializable(KEY_AUTH_AUTHORIZATION_AGENT, mAuthorizationAgent);
         outState.putString(KEY_AUTH_REDIRECT_URI, mRedirectUri);
         outState.putString(KEY_AUTH_REQUEST_URL, mAuthorizationRequestUrl);
     }
 
+    @Override
+    public void onBackPressed() {
+        Logger.verbose(TAG, "Back button is pressed");
```

Comment:
what happens if they press back on the MFA screen?

Reply:
@addev-ashish 
If the prompt behavior is SELECT_ACCOUNT or CONSENT, it will back to the select account page.
If the prompt behavior is FORCE_PROMPT, it will back to the login page.